cmd/config/profile/delete_test.go

```diff
@@ -0,0 +1,64 @@
+package profile_test
+
+import (
+	"testing"
+
+	"github.com/pingidentity/pingctl/internal/testing/testutils"
+	"github.com/pingidentity/pingctl/internal/testing/testutils_cobra"
+)
+
+// Test Config Profile Delete Command Executes without issue
+func TestConfigProfileDeleteCmd_Execute(t *testing.T) {
+	err := testutils_cobra.ExecutePingctl(t, "config", "profile", "delete", "production")
```

Comment:
Seems a bit odd that add requires a `--name` argument but delete (and possibly other subcommands) just take the last trailing argument as the name

Reply:
This is a good point. I've noticed based on cobra docs (https://github.com/spf13/cobra/blob/371ae25d2c82e519feb48c82d142e6a696fd06dd/command.go#L61) the `Use` attribute of cobra commands can be customized to inform available flags and arguments to reduce confusion. 

Because the profile doesn't exist and the name isn't required to execute the command, I think an optional flag makes sense here.